app/controllers/claims_controller.rb

```diff
@@ -85,6 +85,10 @@ def check_page_is_in_sequence
   end
 
   def page_sequence
-    @page_sequence ||= PageSequence.new(current_claim, params[:slug])
+    @page_sequence ||= PageSequence.new(current_claim, claim_slug_sequence, params[:slug])
+  end
+
+  def claim_slug_sequence
+    current_claim.eligibility.class.parent::SlugSequence.new(current_claim)
```

Comment:
Could we break out the bit where we get the class out into a method? (or have is as a variable) Just to be clearer about the intention here. Something like:

```
def eligibility_context
  current_claim.eligibility.class.parent
end

def claim_slug_sequence
  eligibility_context::SlugSequence.new(current_claim)
end
```

Reply:
Yeah, I've toyed around with naming this thing and pulling it out to a method but feel like until it's being used in more places, it might not be the best time to do it. I like 'eligibility context'. I was also thinking of putting it on the claim itself, that way it's available in more places, e.g. `current_claim.policy_namespace`. Can we delay doing this for now? I think it will happen as the rest of the M&P refactor work continues.

Reply:
Ah yeah, cool. I like the idea of putting it in the model too. Happy to leave it for now in that case 👍